packages/flutter/lib/src/cupertino/refresh.dart

```diff
@@ -249,7 +249,10 @@ typedef RefreshCallback = Future<void> Function();
 ///
 /// Can only be used in downward-scrolling vertical lists that overscrolls. In
 /// other words, refreshes can't be triggered with lists using
-/// [ClampingScrollPhysics].
+/// [ClampingScrollPhysics]. In order for [CupertinoSliverRefreshControl] to work properly
+/// on Android (which uses [ClampingScrollPhysics] by default), be sure to override
```

Comment:
Be direct in documentations and avoid 'editorializing'. Just 'override' instead of 'be sure to override'.

They can override physics or insert different ScrollConfiguration in the tree above it as well. 

Add a see also to ScrollConfiguration in the doc list below too.